src/test/java/com/puppycrawl/tools/checkstyle/checks/blocks/RightCurlyCheckTest.java

```diff
@@ -84,6 +84,16 @@ public void testSameOmitOneLiners() throws Exception {
         verify(checkConfig, getPath("InputRightCurlyNameForOneLiners.java"), expected);
     }
 
+    @Test
+    public void testSameDoesNotComplainForNonMultilineConstructs() throws Exception {
+        final DefaultConfiguration checkConfig = createModuleConfig(RightCurlyCheck.class);
+        checkConfig.addAttribute("option", RightCurlyOption.SAME.toString());
+        checkConfig.addAttribute("tokens", "LITERAL_DO, LITERAL_FOR, LITERAL_WHILE, STATIC_INIT,"
+                + "INSTANCE_INIT, CLASS_DEF, METHOD_DEF, CTOR_DEF");
+        final String[] expected = CommonUtil.EMPTY_STRING_ARRAY;
+        verify(checkConfig, getPath("InputRightCurlySame.java"), expected);
```

Comment:
Do we have enough examples for `same` that will show violations for all the items you added in this input?
None of the other inputs changed in violations, which makes me think we may not have enough examples.

Reply:
I'm not entirely sure what you mean here.
In the file are the cases that I reported, that failed without my fixes.

Reply:
We had no tests where good code turned into violations or violations turned into no violation.
All the cases you added are good code. There is no new violation cases we are missing from your changes?

Reply:
I didn't add new violations, I just removed wrong violations that were not tested, so why should any existing test change? 😕

Reply:
Basically you changed the logic of the check. If you ran regression now, you would see a whole bunch of differences in the report. Seeing no differences in our test suite means we don't have enough examples to replicate a mini-regression. If we only went by test suite instead of regression, we would think this change caused no differences.

We are going to ask for regression report for this issue anyways. Why don't you run it, and if anything is now causing a violation add a mini-version of it to our test suite as an example so we have more examples next time.

Reply:
I'm still not sure I get you.

All differences the regression test will find (running it currently) will be false positives that were found before and are not found after.

There **is** a difference in the test suite, I added a new test that is red without my changes and green with my changes, this false-positive case just was not tested before which is the reason there are no changes in existing tests.